src/Core/Resolvers/SqlPaginationUtil.cs

```diff
@@ -577,12 +578,14 @@ public static JsonElement CreateNextLink(string path, NameValueCollection? query
         /// </summary>
         /// <param name="jsonResult">Results plus one extra record if more exist.</param>
         /// <param name="first">Client provided limit if one exists, otherwise 0.</param>
+        /// <param name="Default">Default limit if none provided.</param>
         /// <returns>Bool representing if more records are available.</returns>
-        public static bool HasNext(JsonElement jsonResult, uint? first)
+        public static bool HasNext(JsonElement jsonResult, int? first, uint defaultPageSize = PaginationOptions.DEFAULT_PAGE_SIZE, uint maxPageSize = PaginationOptions.MAX_PAGE_SIZE)
```

Comment:
is this logic being repeated in src/Core/Resolvers/SqlResponseHelpers.cs? Could just keep either version